src/components/blocks/SearchResult.tsx

```diff
@@ -1,69 +1,52 @@
-import { convertDate } from "@utils/timedate";
 import { DocumentListItem } from "@components/document/DocumentListItem";
-import MatchesButton from "@components/buttons/MatchesButton";
-import { TDocument } from "@types";
+import { FamilyListItem } from "@components/document/FamilyListItem";
+import { TFamilyDocument, TFamily } from "@types";
 
 interface SearchResultProps {
-  document: TDocument;
+  family: TFamily;
 }
 
-const SearchResult = ({ document }: SearchResultProps) => {
-  const {
-    document_geography,
-    document_postfix,
-    document_slug,
-    document_date,
-    document_description,
-    document_name,
-    document_category,
-    document_title_match,
-    document_description_match,
-    document_passage_matches,
-    document_content_type,
-  } = document;
-
-  const formatDate = () => {
-    const eudate = document_date;
-    const dateArr = eudate.split("/");
-    return `${dateArr[1]}/${dateArr[0]}/${dateArr[2]}`;
-  };
-  const [year] = convertDate(formatDate());
+const SearchResult = ({ family }: SearchResultProps) => {
```

Comment:
Dunno if you want "Grouped" somewhere in this name - as the search response differs - see [here](https://github.com/climatepolicyradar/navigator-backend/blob/3b82e0eeec74fe8b784aa7511d2441d2f8c98d4d/app/api/api_v1/routers/search.py#L80) and [here](https://github.com/climatepolicyradar/navigator-backend/blob/3b82e0eeec74fe8b784aa7511d2441d2f8c98d4d/app/api/api_v1/routers/search.py#L106)

Reply:
Good suggestion to catch the difference in API response - but this should be handled in the `useSearch` hook instead of a component.